src/styles/index.scss

```diff
@@ -1,3 +1,125 @@
+@import 'variables';
+@import 'stars';
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
```

Comment:
Consider importing Google Fonts in your HTML file. It's a best practice to include external resources in your HTML file to reduce the number of HTTP requests.